src/source.rs

```diff
@@ -106,8 +106,6 @@ pub trait Locker: fmt::Debug {
 pub enum ResolveResponse {
   /// The resolved specifier where the module is an AMD module.
   Amd(ModuleSpecifier),
```

Comment:
@kitsonk do you think it's worth still having Amd, SystemJs, Umd?

Reply:
🤷 I still think it is worth having it support CommonJS, as it makes it more general purpose.  Just because Deno CLI doesn't use it anymore doesn't make it useful.